docs/data-sources/data-source--json.md

```diff
@@ -4,23 +4,37 @@
 
 ### Data Sources
 
+#### JSON Data
 
-> :rotating_light::rotating_light::rotating_light: **This documentation has not been written yet.** :rotating_light::rotating_light::rotating_light:
-> If you have a desire to contribute back to the Contentment package project, (and you have the time), please consider helping me write the documentation pages for the data-sources.
-> Any discussion, guidance or feedback can be had on issue #145.
-> https://github.com/leekelleher/umbraco-contentment/issues/145
+This data-source enables you to enter the URL of a JSON data source, using it to populate the items of a compatible editor, e.g. [Data List](../editors/data-list.md).
 
+##### How to configure the editor?
 
-#### JSON Data
+The configuration of the JSON data-source has the following options:
 
-Configure JSON data to populate the data source.
+![image](https://user-images.githubusercontent.com/85704521/156576985-6a4ce1d8-6888-4392-8e07-4eaa0d660316.png)
 
+The first field is URL, here you can enter either a remote URL, or a local relative file path.
 
-##### How to configure the editor?
+e.g. `https://leekelleher.com/umbraco/contentment/data.json` or `~/umbraco/config/lang/en.json`
+
+<!--- 1- you must only use a URL that have a domain of the JSON data source not a relative URL (true?)--->
```

Comment:
That's correct, if the value starts with `http` then it's treated as a web request, otherwise treated as a local file path.